### DISCUSSION TITLE
Grid view shows manual and scheduled flows not in order of date/time

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#29101] [Q&A] [answered]
When looking at the grid view of a flow, the manual and scheduled runs don't get shown in order of date/time.
Is there a setting i must adjust for this?
we are using airflow Version: [v2.3.4]

**Comment by potiuk** [comment, marked as answer]
It's a bug that has been fixed already in one of the later versions (see the changelog and find it). 

Generally it's a good idea to always update to latest available airflow version (2.5.1 as of last week) - as every single patchlevel release contain many tens of bugfixes.

Just upgrade.
